### DISCUSSION TITLE
Could you please add Xatab

### DISCUSSION THREAD

**Discussion by ghost** [iMAboud/iMSteam#9] [Ideas]
https://byxatab.com/

**Comment by iMAboud** [comment]
This site doesn't use an indexer to search for a query, it filters out content. 
The method I'm using is, to go to a site, search for something and check the link for that quarry, this will give me the link which should be used in this script. 

However, if you need to add any other sites which supports querries, you can follow these steps using (Tampermonkey user-script) which I just updated today. 

- Go to any site e.g: https://rutracker.org/
- Do a search, e.g: The Witcher
- Notice the link changes to: https://rutracker.org/forum/tracker.php?nm=the+witcher
- Remove "the+witcher" to be like this: https://rutracker.org/forum/tracker.php?nm=
- Then add it to the user-script like this: 

    const rutracker= `https://rutracker.org/forum/tracker.php?nm=${formattedGameName}`;
    createButton(rutracker, "RuTracker", "Search on RuTracker", "URL_FOR_ICON.PNG");
    
(Make sure to use this exact format, and add its name in lower case for IDs e.g (rutacker), also add a png icon link).

**Reply by iMAboud** [reply]
const rutracker= `https://rutracker.org/forum/tracker.php?nm=${formattedGameName}`;
    createButton(rutracker, "RuTracker", "Search on RuTracker", "URL_FOR_ICON.PNG");
